youtube.yaml

```diff
@@ -2,5 +2,9 @@
 channels:
   - offered_by: MITx
     channel_id: UCTBMWu8yshnAmpzR3MoJFtw
+    playlists:
+      - UCTBMWu8yshnAmpzR3MoJFtw
```

Comment:
This should be UUTBMWu8yshnAmpzR3MoJFtw

https://developers.google.com/youtube/v3/docs/channels/list?apix_params=%7B%22part%22%3A%22contentDetails%22%2C%22id%22%3A%22UCTBMWu8yshnAmpzR3MoJFtw%22%7D

Reply:
The other playlist id is off by one letter too.
I put up a fixed pr:
https://github.com/mitodl/open-video-data/pull/4